cmd/nvidia-dra-plugin/driver.go

```diff
@@ -26,7 +26,7 @@ import (
 	"k8s.io/apimachinery/pkg/watch"
 	"k8s.io/client-go/util/retry"
 	"k8s.io/klog/v2"
-	drapbv1 "k8s.io/kubelet/pkg/apis/dra/v1alpha2"
+	drapbv1 "k8s.io/kubelet/pkg/apis/dra/v1alpha3"
```

Comment:
I didn't think we could make this change until we also made the `kind` changes. My expectation was that this PR would pull in the controller changes from #14 but not the kubelet-plugin changes. I'd also prefer to see this as a sparate commit, not bundled in the same commit as the go.mod changes.